cli/src/cmd/forge/script/mod.rs

```diff
@@ -175,6 +175,10 @@ pub struct ScriptArgs {
     #[clap(long)]
     pub slow: bool,
 
+    /// Disables interactive prompts that might appear when deploying big contracts.
```

Comment:
can we add info that this is EIP-170 contract size limit

Reply:
added in https://github.com/foundry-rs/foundry/pull/5590/commits/72ce4d82ced1752c3e554dc893dbd11ccd7a6e22